spring-xd-reactor/src/main/java/org/springframework/xd/reactor/BroadcasterMessageHandler.java

```diff
@@ -65,18 +67,19 @@
  * @author Mark Pollack
  * @author Stephane Maldini
  */
-public class BroadcasterMessageHandler extends AbstractMessageProducingHandler  implements DisposableBean {
+public class BroadcasterMessageHandler extends AbstractMessageProducingHandler implements DisposableBean {
 
     protected final Log logger = LogFactory.getLog(getClass());
 
-    private final RingBufferProcessor<Object> stream;
-
-    @SuppressWarnings("rawtypes")
-    private final Processor reactorProcessor;
+    private final RingBufferProcessor<Object> ringBufferProcessor;
 
     private final Class<?> inputType;
 
-    private Subscription processorSubscription;
+    private int stopTimeout = 5000;
+
+    private int ringBufferSize = 8192;
```

Comment:
will create AbstractReactorMessageHandler to ensure consistent handling and no simple code dup for environment, stopTimeout, and ringBuffer fields.